tests/test_sourmash.py

```diff
@@ -4661,6 +4661,35 @@ def test_gather_output_unassigned_with_abundance(runtmp, prefetch_gather, linear
             assert nomatch_mh.hashes[hashval] == abund
 
 
+def test_multigather_output_unassigned_with_abundance(runtmp):
+    c = runtmp
+    query = utils.get_test_data('gather-abund/reads-s10x10-s11.sig')
+    against = utils.get_test_data('gather-abund/genome-s10.fa.gz.sig')
+
+    with pytest.raises(SourmashCommandFailed):
```

Comment:
Here you are calling multigather with incorrect arguments, which is why the command fails. Please inspect the output of `sourmash multigather -h` or the implementation in `commands.py` to see if you can figure out what the right arguments are.

Reply:
Alright, I'll do that. Thanks!

Reply:
I can't seem to figure out which arguments would be appropriate for this command. I didn't find any equivalent argument for `--output-unassigned` when I did `sourmash multigather -h`. I was also thinking of using `--query-from-file`. I'm not sure if that's correct though.